Update data analysis with main

christiebarron opened the pull request.

christiebarron committed:
Merge pull request #27 from christiebarron/dataAnalysis

Data analysis

christiebarron committed:
Update README.md

christiebarron committed:
Rename projectProposal.md to projectProposal1.md

christiebarron committed:
Create project2Proposal

christiebarron committed:
Update and rename project2Proposal to project2Proposal.md

christiebarron committed:
add AI web scrape

christiebarron committed:
Merge branch 'main' of github.com:christiebarron/AITextDetector

christiebarron committed:
revise ai text scrape file organization

christiebarron committed:
add draft sql database ERD

christiebarron committed:
reorganize ERD files

birdipa2 committed:
Project 2 Transformation

birdipa2 committed:
Merge pull request #30 from christiebarron/dataTransformation

Project 2 Transformation

GayanMeerigama commented:
[speedypaper.csv](https://github.com/christiebarron/aiTextDetector/files/11484175/speedypaper.csv)

GayanMeerigama committed:
Add files via upload

christiebarron committed:
added mock csvs for the database design

christiebarron committed:
Create sqlAlchemy.ipynb

christiebarron committed:
revised database CSVs

GayanMeerigama committed:
Add files via upload

GayanMeerigama committed:
Delete speedypaper.ipynb

GayanMeerigama committed:
Add files via upload

GayanMeerigama committed:
Add files via upload

christiebarron committed:
Merge pull request #31 from christiebarron/main

pull updates from main

christiebarron committed:
Merge pull request #32 from christiebarron/dataTransformation

Data transformation

GayanMeerigama committed:
Add files via upload

vareesha7 committed:
new

vareesha7 committed:
Merge branch 'main' of https://github.com/christiebarron/aiTextDetector

christiebarron committed:
revise folder structure

christiebarron committed:
revise conflict regarding web scrape

christiebarron committed:
documentation and folder structure revision

birdipa2 committed:
Added PK to every table (for review)

birdipa2 committed:
Added PK to every table (for review)

birdipa2 committed:
SQL DB Schema Structure (Draft for review)

vareesha7 committed:
Add files via upload

GayanMeerigama committed:
Add files via upload

christiebarron committed:
Create project2Report.md

christiebarron committed:
revise proposal and report

christiebarron committed:
Update project2Report.md

christiebarron committed:
Update project2Report.md

christiebarron committed:
organize files

christiebarron committed:
Merge branch 'main' of github.com:christiebarron/AITextDetector

maisamalam committed:
Add files via upload

Maisams upload!

christiebarron committed:
Integrate Maisam's upload into file organization

christiebarron committed:
Update project2Report.md

christiebarron committed:
Update project2Report.md

christiebarron committed:
Update project2Report.md

christiebarron committed:
Update README.md

christiebarron committed:
Create project3Proposal.md

christiebarron committed:
re-format project3 proposal

christiebarron committed:
Update project3Proposal.md

maisamalam committed:
the website

This is a initial commit of the website. At this point, the website has:
- Login page
- Second page
-JS file
-Flask as a backend that pulls from the db on render.com
-Also very basic css.

maisamalam committed:
the website

This is a initial commit of the website. At this point, the website has:
- Login page
- Second page
-JS file
-Flask as a backend that pulls from the db on render.com
-Also very basic css.

christiebarron committed:
preliminary flask and plotly work

christiebarron committed:
Create project3Proposal.md

christiebarron committed:
created mock data for project 3

christiebarron committed:
Merge branch 'main' into Database-branch

christiebarron committed:
Merge pull request #34 from christiebarron/Database-branch

Database branch

maisamalam committed:
login page finally works!

birdipa2 committed:
Added files for comparative analysis

The data fetching is still giving an error

birdipa2 committed:
added xls file for app.py

maisamalam committed:
website

maisamalam committed:
Merge branch 'main' of https://github.com/christiebarron/aiTextDetector

maisamalam committed:
ai update

GayanMeerigama committed:
Add files via upload

vareesha7 committed:
Create Word cloud

word cloud

vareesha7 committed:
Delete Word cloud

vareesha7 committed:
Create student essay

vareesha7 committed:
Add files via upload

vareesha7 committed:
Delete word cloud.js

vareesha7 committed:
Add files via upload

GayanMeerigama committed:
Add files via upload

christiebarron committed:
add files from data-transformation branch

christiebarron committed:
add file from data-transformation branch

birdipa2 committed:
updated backend

birdipa2 committed:
Update style.css

birdipa2 committed:
Update script_data.js

birdipa2 committed:
Add files via upload

vareesha7 committed:
Delete word cloud.js

vareesha7 committed:
Add files via upload

GayanMeerigama committed:
Delete app.py

GayanMeerigama committed:
Add files via upload

birdipa2 committed:
background.jpg

christiebarron committed:
revise back to original

birdipa2 committed:
Add files via upload

birdipa2 committed:
Update README.md

GayanMeerigama committed:
Add files via upload

GayanMeerigama committed:
Delete KNN-HumanVsHuman directory

GayanMeerigama committed:
Add files via upload

GayanMeerigama committed:
Delete KNN-HumanVsHuman directory

GayanMeerigama committed:
Add files via upload

christiebarron committed:
Add files via upload

GayanMeerigama committed:
Add files via upload

christiebarron committed:
Merge branch 'main' of github.com:christiebarron/AITextDetector

christiebarron committed:
add generated ai text

christiebarron committed:
Update project4Proposal.md

add Paramdeep's descriptions

christiebarron committed:
Update project4Proposal.md

maisamalam committed:
Update project4Proposal.md

vareesha7 committed:
Update project4Proposal.md

birdipa2 committed:
Update project4Proposal.md

GayanMeerigama committed:
Update project4Proposal.md

christiebarron committed:
Create 8aNlpPipelineFlask.ipynb

draft of code to automatically score text as AI-generated

GayanMeerigama committed:
Add files via upload

christiebarron committed:
revised NLP pipeline for flask

christiebarron committed:
Merge branch 'main' of github.com:christiebarron/AITextDetector

christiebarron committed:
read new data, add to existing excel documents

christiebarron committed:
revised 3bPreprocessData script to work as intended

birdipa2 committed:
AI vs. Human Essay Classifier

Features:

Preprocessing of essays to clean and tokenize the text.
Extracting relevant features such as word frequency, sentence length, and more.
Applying Min-Max scaling to ensure features are within the same range.
Selecting the top k features using chi-squared scoring.
Training a Random Forest Classifier to classify essays.
Hyperparameter tuning using GridSearchCV for optimal model performance.
Evaluating multiple classifiers (Random Forest, SVM, and Logistic Regression).
Generating classification reports to analyze precision, recall, and F1-score.
Handling misclassified instances.

vareesha7 committed:
Create f

vareesha7 committed:
Update and rename f to Files

vareesha7 committed:
Delete primary/Visualizations directory

vareesha7 committed:
Create Files

vareesha7 committed:
Add files via upload

GayanMeerigama committed:
Delete AI_Website3/Dashapp_text_input directory

GayanMeerigama committed:
Add files via upload

christiebarron committed:
Update 4aFeature ExtractionPipeline.ipynb

christiebarron committed:
revise git ignore to ignore big excel file

christiebarron committed:
add feature extraction to app.py

birdipa2 committed:
Tentative Code for Flask App

birdipa2 committed:
Keras Tuner Model

birdipa2 committed:
Updated Flask App

maisamalam committed:
Add files via upload

vareesha7 committed:
Add files via upload

vareesha7 committed:
Create tableau

vareesha7 committed:
Add files via upload

birdipa2 committed:
Tuned Keras Model

birdipa2 committed:
Updated with Keras Tuner Model

birdipa2 committed:
Keras_Tuner_Model

birdipa2 committed:
Updated Index

birdipa2 committed:
prediction for Keras html

birdipa2 committed:
Result_Keras.html

GayanMeerigama committed:
Add files via upload

birdipa2 committed:
FINAL PROJECT
